packages/ui/src/components/List/constants.tsx

```diff
@@ -1 +1,2 @@
 export const SELECTABLE_CHECKBOX_SIZE = 24
+export const EXPAND_CTA_SIZE = 24
```

Comment:
You can rename this one too 

```suggestion
export const EXPAND_BUTTON_SIZE = 24
```